git-multimail/git_multimail.py

```diff
@@ -2012,11 +2012,10 @@ def find_sendmail():
         for path in SendMailer.SENDMAIL_CANDIDATES:
             if os.access(path, os.X_OK):
                 return path
-        else:
-            raise ConfigurationException(
-                'No sendmail executable found.  '
-                'Try setting multimailhook.sendmailCommand.'
-                )
+        raise ConfigurationException(
```

Comment:
In both of these cases in this commit I prefer the original code because it alerts the reader that there is probably another `return` statement somewhere in the function.

Reply:
indeed, probably another false positive that could be reported to lgtm.

Reply:
Yeah, they have examples of this vs the earlier code for the rule, but I struggled to understand why one was objectively better than the other.